### PR TITLE
Archive Update 53.1

### PR DESCRIPTION
**Pull request by Tschrock**
Latest changes from the custom server archive download for PonyTown update 0.53.1

### CODE REVIEW

File: src/ts/components/shared/play-notice/play-notice.pug [hunk 1]
```diff
@@ -2,3 +2,9 @@
 	h5 General rules
 	ul.text-muted.list-rules
 		li(*ngFor="let r of rules") {{r}}
+
+	h5 Notice
+	p.text-muted
+		| This game is very #[strong early in development]. There might be bugs and occasional downtimes.
+	p.text-muted
+		| Please do not redistribute any of the game files or code.
```
Comment (Stubenhocker1399) [inline src/ts/components/shared/play-notice/play-notice.pug]: This seems like an accidental addition.

Comment (LunaSquee) [reply src/ts/components/shared/play-notice/play-notice.pug]: If we're not supposed to redistribute the code, why is it using such a permissive license without restrictions? This has to be a mistake.

Comment (Tschrock) [reply src/ts/components/shared/play-notice/play-notice.pug]: This is the text that shows up on the main site, - if I had to guess, I would say it was inadvertently included with the custom server archive.

**Comment by Tschrock** [comment]
I've created an `archive` branch to track update changes and have reworked this PR as a merge from that. 
This makes things a lot easier since git can tell you where the conflicts are, etc.

I've looked over things, and nothing has changed from my original PR other than the normalization of BOM/Line endings.

File: src/ts/components/shared/play-notice/play-notice.pug (hunk 1 above)
Comment (LunaSquee) [reply src/ts/components/shared/play-notice/play-notice.pug]: I had reached out for clarification, it is indeed a mistake.
https://termbin.lunasqu.ee/26bt

**Pull request merged**